packages/edit-site/src/components/site-hub/index.js

```diff
@@ -77,9 +81,16 @@ const SiteHub = memo(
 								variant="link"
 								href={ homeUrl }
 								target="_blank"
-								label={ __( 'View site (opens in a new tab)' ) }
+								icon={ external }
+								iconPosition="right"
 							>
 								{ decodeEntities( siteTitle ) }
+								<VisuallyHidden as="span">
+									{
+										/* translators: accessibility text */
+										__( 'View site (opens in a new tab)' )
```

Comment:
```suggestion
										__( '(opens in a new tab)' )
```

For consistency with other occurrences I'd remove the 'View site' part.